docs/data/registry.json

```diff
@@ -1037,23 +1037,25 @@
     "type": "solr",
     "versions": {
       "deprecated": [
+        "8.6",
+        "8.4",
+        "8.0",
+        "7.7",
         "7.6",
         "6.6",
         "6.3",
         "4.10",
         "3.6"
       ],
       "supported": [
-        "8.11",
-        "8.6",
-        "8.4",
-        "8.0",
-        "7.7"
+        "8.11"
       ]
     },
     "versions-dedicated": {
       "supported": [
-        "8.11",
+        "8.11"
+      ],
+      "deprecated":[
```

Comment:
:lady_beetle:
8.6 and 6.6 for Dedicated Generation 3 should be moved to deprecated.
since [<8.11 - All older versions are End Of Life (EOL)](https://solr.apache.org/downloads.html)